packages/panzer/adapters-stk/test/local_mesh/local_mesh_utilities.cpp

```diff
@@ -81,14 +81,14 @@ TEUCHOS_UNIT_TEST(localMeshUtilities, basic)
   // Make sure if fails when you pass in a null mesh
   {
     panzer_stk::STK_Interface mesh;
-    panzer::LocalMeshInfo<int,int> mesh_info;
+    panzer::LocalMeshInfo<int,panzer::Ordinal64> mesh_info;
```

Comment:
Will we be removing this templating at the same time as the DOFManager? I don't think any downstream codes directly access the LocalMeshInfo objects, so it should be pretty clean.

Reply:
yes - I might do that before starting the DOF manager